From 2df0d625ba8ebb28f5c4702bef096a917830278f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20Hor=C3=A1k?= <dan[at]danny.cz>
Date: Wed, 18 Jan 2012 20:41:42 +0100
Subject: [PATCH 07/11] install FlightCrew library and headers

With minor changes by Hans de Goede:
-Also install the headers under src/FlightCrew/Misc and
 src/FlightCrew/Validators
---
 src/FlightCrew/CMakeLists.txt |   14 ++++++++++++++
 1 files changed, 14 insertions(+), 0 deletions(-)

diff --git a/src/FlightCrew/CMakeLists.txt b/src/FlightCrew/CMakeLists.txt
index e86fc19..9214c85 100644
--- a/src/FlightCrew/CMakeLists.txt
+++ b/src/FlightCrew/CMakeLists.txt
@@ -135,3 +135,17 @@ endif()
 if( NOT ( BUILD_SHARED_FC AND MSVC ) AND NOT NO_TEST_EXE )
     add_subdirectory( tests )
 endif()
+
+if( UNIX AND NOT APPLE )
+    install( TARGETS ${PROJECT_NAME}
+        LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}
+     )
+    file( GLOB files "${CMAKE_CURRENT_SOURCE_DIR}/*.h" )
+    install( FILES ${files} DESTINATION ${INCLUDE_INSTALL_DIR}/${PROJECT_NAME} )
+    file( GLOB files "${CMAKE_CURRENT_SOURCE_DIR}/Misc/*.h" )
+    install( FILES ${files} DESTINATION ${INCLUDE_INSTALL_DIR}/${PROJECT_NAME}/Misc )
+    file( GLOB files "${CMAKE_CURRENT_SOURCE_DIR}/Validators/*.h" )
+    install( FILES ${files} DESTINATION ${INCLUDE_INSTALL_DIR}/${PROJECT_NAME}/Validators )
+    install( FILES ${CMAKE_CURRENT_SOURCE_DIR}/../XercesExtensions/NodeLocationInfo.h DESTINATION ${INCLUDE_INSTALL_DIR}/${PROJECT_NAME} )
+endif()
-- 
1.7.7.4

